documentation/design-docs/dotnet-tools.md

```diff
@@ -447,52 +436,49 @@ COLLECT
 
 
      Examples:
-       > dotnet-dump collect --process-id 1902 --number 2 --type mini --output ~/dumps/go/here/
-       Writing:     ~/dumps/go/here/core0001
-    
-... 10 seconds pass (the default time interval)
-
-       > dotnet-dump collect --process-id 1902 --number 2 --type mini --output ~/dumps/go/here/
-       Writing:     ~/dumps/go/here/core0001
-       Writing:     ~/dumps/go/here/core0002
-       Complete
 
+        $ dotnet dump collect --process-id 1902 --number 2 --type mini
+        Writing minidump to file ./core_20190226_135837
+        Written 98983936 bytes (24166 pages) to core file
+        Complete
+        
+        $ dotnet dump collect --process-id 1902 --number 2 --type mini
+        Writing minidump to file ./core_20190226_135850
+        Written 98959360 bytes (24160 pages) to core file
+        Complete
 
 ANALYZE
 
     dotnet-dump analyze [-h|--help] dump_path
 
     Starts an interactive shell with debugging commands to explore a dump
 
-    -h, --help
-        Show command line help
-
     dump_path
         The dump to analyze
 
     Examples:
-      > dotnet-dump analyze core0002
-      Use 'help' for help, 'q' to quit
-      $
+      $ dotnet-dump analyze ./core_20190226_135850
+      Loading core dump: ./core_20190226_135850
+      >
```

Comment:
I think some form of help text would be nice at the start no? I'm open to what that is, but its good to know how to get more help and how to exit back to the shell.

Reply:
What about:
```
Ready to process analysis commands. Type 'help' to list available commands or 'help [command]' to get detailed help on a command.
Type 'exit' to exit the analysis session.

```